gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -65,36 +65,30 @@ const NxCardPage = () =>
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card__header</code></td>
-            <td className="nx-cell">Element</td>
+            <td className="nx-cell"><code className="nx-code">.nx-card--horizontal</code></td>
+            <td className="nx-cell">Modifer of <code className="nx-code">.nx-card</code></td>
             <td className="nx-cell">
-              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
+              Used for cards that are laid out horizontally rather than vertically.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card__header</code></td>
             <td className="nx-cell">Element</td>
             <td className="nx-cell">
-              Wraps the card call out (if any). The call out typically consists of a number, sparkline, icon, or
-              other small graphic. It should not be free form text.
+              Card header element, typically wraps an <code className="nx-code">&lt;h3&gt;</code>.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card__call-out--text-only</code></td>
-            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card__call-out</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card__footer</code></td>
+            <td className="nx-cell">Element</td>
             <td className="nx-cell">
-              Used when the content of <code className="nx-code">.nx-card__call-out</code> consists of text.
+              In a vertically oriented card the footer. Typically provides a link to more information.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card__text</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card__actions</code></td>
```

Comment:
Should `.nx-card__footer` be merged with this?  It seems like they do the same things just in the two different kinds of cards.

Reply:
> Should `.nx-card__footer` be merged with this? It seems like they do the same things just in the two different kinds of cards.

I purposely separated them because in the horizontal layout they appear at the right side, not below. They had so little in common I just didn't see the point.

Reply:
> I purposely separated them because in the horizontal layout they appear at the right side, not below. They had so little in common I just didn't see the point.

To me they're conceptually the same, and it would be one less class to deal with.  Per your point though, if they were merged I'd say they should be called `.nx-card__actions` not `.nx-card__footer`

Reply:
> > I purposely separated them because in the horizontal layout they appear at the right side, not below. They had so little in common I just didn't see the point.
> 
> To me they're conceptually the same, and it would be one less class to deal with. Per your point though, if they were merged I'd say they should be called `.nx-card__actions` not `.nx-card__footer`

https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/307a84fed051a9b3fb6edaf35150c47d3a22fa65